entity-framework/core/what-is-new/ef-core-7.0/plan.md

```diff
@@ -0,0 +1,342 @@
+
+Much of the work in this theme overlaps with work already outlined above. Some of the more significant work items are:
+
+- [Issue #214: Expose model building conventions to applications](https://github.com/dotnet/efcore/issues/214)
+- [Issue #245: Map inserts, updates, and deletes (CUD operations) to stored procedures](https://github.com/dotnet/efcore/issues/245)
+- [Issue #620: Entity splitting support](https://github.com/dotnet/efcore/issues/620)
+- [Issue #3170: TPC inheritance mapping pattern](https://github.com/dotnet/efcore/issues/3170)
+- [Issue #3864: Support unidirectional many-to-many relationships through shadow navigations](https://github.com/dotnet/efcore/issues/3864)
+- [Issue #4038: Code templates for scaffolding entity types and DbContext from an existing database](https://github.com/dotnet/efcore/issues/4038)
+- [Issue #10753: Support raw SQL queries without defining an entity type for the result](https://github.com/dotnet/efcore/issues/10753)
+- [Issue #11003: KeyAttribute support for composite primary key](https://github.com/dotnet/efcore/issues/11003)
+- [Issue #11624: Support raw SQL Queries for basic types like Guid, DateTime and int](https://github.com/dotnet/efcore/issues/11624)
+- [Issue #15911: Implement ObjectMaterialized event](https://github.com/dotnet/efcore/issues/15911)
+- [Issue #17653: Support for GroupBy entityType](https://github.com/dotnet/efcore/issues/17653)
+- [Issue #19929: Support GroupBy when it is final operator](https://github.com/dotnet/efcore/issues/19929)
+- [Issue #19930: Support GroupJoin when it is final query operator](https://github.com/dotnet/efcore/issues/19930)
+- [issue #26626: Use EF6 as an oracle for procedural testing of EF Core queries](https://github.com/dotnet/efcore/issues/26626)
+
+In addition, we plan to make it clear on the [legacy EF6 GitHub repo](https://github.com/dotnet/ef6) that we are not planning any future work on EF6. The exception to this is that we plan to add support for using EF6 with [Microsoft.Data.SqlClient](https://github.com/dotnet/SqlClient). This will be limited to runtime support. Use of the EF6 designer in Visual Studio will still require `System.Data.SqlClient`.
+
+Note that EF Core has a fundamentally different architecture to EF6. In particular, it does not make use of an Entity Data Model (EDM) specification. This means certain features such as EDMX and EntitySQL will never be supported in EF Core.
+
+## Theme: Performance
+
+Great performance is a fundamental tenet of EF Core, lower-level data access, and indeed all of .NET. Every release includes significant work on improving performance. As always, this theme will involve a lot of iterative investigation, which will then inform where we focus resources.
+
+### Performance of database inserts and updates
```

Comment:
Deletions as well, right? The bulk-set work item is about updates and deletes only (no bulk insert), but the general optimization of the update pipeling + change tracking should help all three.